### PR TITLE
Load forking context from config

### PR DESCRIPTION
**Pull request by rouzwawi**
Suggested fix for #103

### CODE REVIEW

**Comment by rouzwawi** [comment]
@danielnorberg @honnix Have I understood things correctly here; the way `TracingContext` and `StructuredLoggingEncoder` are set up (with `Context.key("task-id")`) requires the tasks to run in separate JVMs so that other worker-threads do not concurrently also set that same `Context.key`.

**Comment by danielnorberg** [comment]
How about working on fixing the serialization/forking issues instead? We want to fork in order to have per task logging.

File: flo-runner/src/main/java/com/spotify/flo/context/ForkingContext.java [hunk 1]
```diff
@@ -26,55 +26,23 @@
...
-    if (forking.isPresent()) {
-      if (forking.get()) {
-        log.debug("Forking enabled (environment variable FLO_FORKING=true)");
-        return new ForkingContext(baseContext);
-      } else {
-        log.debug("Forking disabled (environment variable FLO_FORKING=true)");
-        return baseContext;
-      }
-    } else if (inDebugger) {
-      log.debug("Debugger detected, forking disabled by default "
```
Comment (danielnorberg) [inline flo-runner/src/main/java/com/spotify/flo/context/ForkingContext.java]: I think we'd want to preserve the behavior or disabling forking by default if running in the debugger.

**Comment by rouzwawi** [comment]
@danielnorberg Please have another look. The PR preserves the previous behaviour when in the debugger and has forking enabled by default. But allows it to be configured via a `flo.conf` file in user projects.

**Comment by danielnorberg** [comment]
👍

**Pull request merged**